mockOSApi.v2/Controllers/ProcessController.cs

```diff
@@ -81,6 +116,20 @@ public ActionResult<MockProcessDto> Get(int pid)
         return Ok(procs);
 
     }
+    [HttpGet("user/{user}")]
```

Comment:
So this returns the processes under an user?
Try something like `[HttpGet("processes/user/{user}")]`. Never hurts to be more verbose

Reply:
No, this returns the process running <cmdline>, e.g. to return all processes running bash or notepad.exe ;)